docs/src/development/local/lando.md

```diff
@@ -4,43 +4,190 @@ sidebarTitle: "Lando"
 
-At this time, Lando supports only PHP-based applications but works with any service supported by Platform.sh.  Support for other application languages is in progress.
+At this time, Lando is currently in beta and supports only PHP-based applications.
+Support for other application languages is in progress.
+Lando works with any service supported by Platform.sh.
 
-See the [Lando documentation](https://docs.lando.dev/platformsh/) for installing and setting up Lando on your system.
-A quick-start guide is included below, but the Lando documentation is the primary source of truth.
+A quick-start guide is included below, but the [Lando documentation](https://docs.lando.dev/config/platformsh.html) is the primary source of truth.
 
-## Quick start
+## Before you begin
 
-First, ensure that you have the Platform.sh CLI installed and that you have authenticated your account.  Then, [install Lando](https://docs.lando.dev/basics/installation.html) and Docker per Lando's installation instructions.
+Lando requires [specific bash variables](https://docs.lando.dev/platformsh/config.html#environment-variables) to be set locally in order to access your project.
+To have these variables set automatically, install the Platform.sh [CLI](../../gettingstarted/introduction/own-code/cli-install.md).
 
-If you do not yet have your project checked out, run:
+Alternatively, you can set the `$PLATFORMSH_CLI_TOKEN` and `$PLATFORMSH_SITE_NAME` environment variables manually.
 
-```bash
+Familiarity with the Lando [CLI commands](https://docs.lando.dev/cli/) is recommended but not necessary.
+
+Depending of your hardware, running the commands below can take a while.
+Check the [hardware requirements](https://docs.lando.dev/getting-started/installation.html#hardware-requirements) before attempting this guide.
+
+## Installation
+
+See the [Lando documentation](https://docs.lando.dev/getting-started/installation.html) for installing and setting up Lando on your system.
+
+### Get started with Lando
+
+The following examples are based on the [PHP template](https://github.com/platformsh-templates/php).
+
+Several stacks are available on Lando (LAMP, LEMP, MEAN).
+If you use a specific stack, adapt the steps below accordingly.
+See the [Lando documentation for all recipes and stacks available](https://docs.lando.dev/).
+
+Lando requires a `.lando.yml` file. The `init` command can generate that file automatically.
+
+The full output of the commands below have been redacted for brevity.
+
+**TODO: fix the Lando init step for both paths + remove the need for the prompt (just add a note) + add stack + add cloning of the PHP base project**
+
+{{< codetabs >}}
+
+---
+title=On a new project
+file=none
+highlight=false
+---
+
+Run `lando init --recipe platformsh --source platformsh` and follow the instructions provided by the interactive prompt.
+
+``` bash
 $ lando init --source platformsh
+
+? Select a Platform.sh account <email address of your account>
```

Comment:
As I mentioned on another PR recently, this isn't the right format for variables. I'm not sure it's necessary at all here, though :shrug: